masonite/response.py

```diff
@@ -87,6 +89,10 @@ def view(self, view, status=200):
 
         if isinstance(view, dict) or isinstance(view, list):
             return self.json(view)
+        else if isinstance(view, Collection):
+            return view.serialize()
+        else if isinstance(view, Model):
+            return view.to_json()
```

Comment:
it's `elif` not `else if` 😉

Reply:
It's Java mind here.. So sorry.. 😃

Reply:
haha 😅

Reply:
Line number 95. It should be `return self.json(view.to_dict())` as you want add JSON headers while returning response